doc-content/enterprise-only/integration/dev-studio-import-local-git-repo-proc.adoc

```diff
@@ -1,12 +1,12 @@
 [id='dev-studio-import-local-git-repo-proc']
 = Importing a local Git repository
-You can import a local git repository to use with Red Hat JBoss Developer Studio.
+You can import a local Git repository to use with Red Hat Developer Studio.
 
 .Prerequisite
-Red Hat JBoss Developer Studio is installed.
+Red Hat Developer Studio is installed.
 
 .Procedure
-. In Red Hat JBoss Developer Studio, select the server from the *Server* tab and click the start icon to start the server.
+. In Red Hat Developer Studio, select the server from the *Server* tab and click the start icon to start the server.
```

Comment:
start icon: Does the word Start appear in or under the icon? If so, bold and cap Start. And then search for 'start icon'. If not, OK as is.

Reply:
The icon does not have the word "Start" in or under the icon, but "Start the server" appears if the user hovers the mouse pointer over the icon. Thoughts?

Reply:
Keeping as-is